### PR TITLE
Add support for adding description to schema

### PR DESCRIPTION
**Pull request by IvanGoncharov**
Implements https://github.com/graphql/graphql-spec/pull/466

### CODE REVIEW

File: src/utilities/__tests__/getIntrospectionQuery-test.js [hunk 1]
```diff
@@ -18,7 +18,19 @@ describe('getIntrospectionQuery', () => {
     );
   });
 
-  it('include "isRepeatable" field', () => {
+  it('include "isRepeatable" field on directives', () => {
```
Comment (leebyron) [inline src/utilities/__tests__/getIntrospectionQuery-test.js]: Is this related to the isRepeatable change? Or just piggybacking in more tests?

Comment (IvanGoncharov) [reply src/utilities/__tests__/getIntrospectionQuery-test.js]: @leebyron Thanks for the review 👍 
Just copy-pasted tests for `isRepeatable` as starting point for testing that schema's description is present but forgot to change them.
I wrote a new test and committed them.
